servant/src/Servant/Utils/Links.hs

```diff
@@ -86,6 +85,8 @@
 --  bad_link under api after trying the open (but empty) type family
 --  `IsElem'` as a last resort.
 module Servant.Utils.Links (
+  module Servant.API.TypeLevel,
```

Comment:
@dredozubov: I don't fully understand your comment. What is shadowed by what here? In any case I would expect the order of the export list to be irrelevant.